src/test/java/core/basesyntax/PasswordValidatorTest.java

```diff
@@ -27,7 +27,7 @@ public void passwordValidate_exceptionClassExists() {
         try {
             Class.forName(EXCEPTION_NAME);
         } catch (ClassNotFoundException e) {
-            Assert.fail("Should create a class called 'PasswordValidationException'.");
+            Assert.fail("Should create a class called 'core.basesyntax.exception.PasswordValidationException'.");
```

Comment:
please look through the changes to the test classes that you made - and return everything back